sipyco/test/test_broadcast.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import asyncio
+
+from sipyco import broadcast
+
+
+test_address = "::1"
+test_port = 7777
+test_message = {"key": "value", "number": 42}
+
+
+class BroadcastCase(unittest.TestCase):
+    def setUp(self):
+        self.loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(self.loop)
+        self.received_messages = []
+        self.message_received = asyncio.Event()
+
+    def notify_callback(self, message):
```

Comment:
nitpick: you can put that function inside ``_do_test_broadcast()`` as it's not used anywhere else (and skip self)